modules/oc-mirror-v2-support.adoc

```diff
@@ -6,13 +6,13 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="oc-mirror-v2-support_{context}"]
-= oc-mirror V2 compatibility and support
+= oc-mirror version 2 compatibility and support
```

Comment:
Again, you use "v2", "version 2" and "V2". I think it would be best to pick one and be consistent throughout